unreleased/hostmetrics-rm-direction-process.yaml

```
@@ -0,0 +1,12 @@
+change_type: deprecation
+
+component: hostmetricsreceiver
+
+note: "Remove direction for paging metrics. The feature gate: `receiver.hostmetricsreceiver.removeDirectionAttributeProcessMetrics` can be set to apply the following"
+
+issues: [11819]
+
+subtext: |2
+    - `process.disk.io` will become:
+      - `process.disk.io.read`
+      - `process.disk.io.write`
```

#11866 ended up padding this text automatically, so the following should be equivalent:
```suggestion
subtext:
  - `process.disk.io` will become:
    - `process.disk.io.read`
    - `process.disk.io.write`

```

---

@djaglowski 

```
FAIL: validate: yaml: line 10: found character that cannot start any token
make: *** [chlog-validate] Error 1
```